post-processing-service/src/main/resources/application.yaml

```diff
@@ -25,7 +25,7 @@ spring:
       javax.persistence.query.timeout: -1
 service:
   fixed-delay:
-    cached-ids: ${FIXED_DELAY_ID:5000}
+    cached-ids: ${FIXED_DELAY_ID:20000}
```

Comment:
This is the important piece of code we need to validate along with the Kafka sink flush time  : 2 seconds
The batch processing delay is 20 seconds.
👍